src/mixins/object_interactivity.mixin.ts

```diff
@@ -230,33 +268,6 @@ export class InteractiveFabricObject extends FabricObject {
     }
   }
 
-  /**
-   * Sets the coordinates that determine the interaction area of each control
-   * note: if we would switch to ROUND corner area, all of this would disappear.
-   * everything would resolve to a single point and a pythagorean theorem for the distance
-   * @todo evaluate simplification of code switching to circle interaction area at runtime
-   * @private
-   */
-  _setCornerCoords(): void {
```

Comment:
refactored into `_calcCornerCoords()` in a revertable commit 5afe4c64f
reasons:
- cleaner/readable code
- better typed
- perf (one less iteration, kept in a contained method since I saw you wrote it might be removed)
- since it is private, considered not breaking